apps/app.go

```diff
@@ -45,11 +45,11 @@ func (v AppVersion) IsValid() error {
 			continue
 		}
 
-		if c == '-' || c == '_' {
+		if c == '-' || c == '_' || c == '.' {
```

Comment:
We shouldn't forbid dots in versions as it's commonly used. `getFunctionName` does sanitize any dots.

Reply:
Developer can have the version "v1.0.0" in the manifest. We will sanitize it later. This method is used to check if the manifest is valid or not. Am I missing something?